### PR TITLE
Cross reference procedures for different videos

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2671](https://openedx.atlassian.net/browse/DOC-2671)

Makes it easier for teams to find info for processing about videos vs. content videos.

### Reviewers
- [x] Subject matter expert: @jaakana
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check): @catong or @pdesjardins 

FYI: @rgrubb

### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### Post-review
- [ ] Squash commits

File: en_us/shared/set_up_course/setting_up_student_view.rst [hunk 1]
```diff
@@ -165,6 +165,10 @@ course content videos.
   on it. If you upload only a course video, the first frame of the the video
   file appears with the **play** icon.
 
+The process for adding a course about video is different than the process for
```
Comment (pdesjardins) [inline en_us/shared/set_up_course/setting_up_student_view.rst]: This section is about course videos that are _not_ the about video, right? Maybe this paragraph should be formatted as a note, to emphasize that it's different than the main subject?

**Comment by pdesjardins** [comment]
Looks good, one comment. :+1:

**Comment by JAAkana** [comment]
Looks good! :palm_tree:

**Pull request merged**